src/interfaces/records/handlers/records-delete.ts

```diff
@@ -66,6 +67,9 @@ export class RecordsDeleteHandler implements MethodHandler {
 
       await this.messageStore.put(tenant, incomingMessage, indexes);
 
+      const messageCid = await computeCid(incomingMessage);
```

Comment:
Nit: consider using `Message.getCid()` for future proofing, even though they end up being the same right now, it hasn't been the case for months when we had `encodedData`, and there is potential that this will be the case again if/when we add `encodedData` (or other properties) back.

Reply:
i've exclusively used `computeCid` everywhere in this PR 😅 . do you want me to go ahead and update all the `computeCid`s i added to `Message.getCid()`?

<img width="404" alt="image" src="https://user-images.githubusercontent.com/4887440/230000717-7dec168a-c37f-4873-9995-c250ae2abb4b.png">

Reply:
I refrained from asking to make the same change in the new tests added, because one can argue that tests should not use product code for validation. But weighing the trade offs, I'd be for changing tests to use `Message.getCid()` also as long as you are okay with it!

Reply:
will replace all instances of `computeCid` in a subsequent PR.

Reply:
will tackle this in a subsequent PR. issue created: #300